src/neptune/types/series/float_series.py

```diff
@@ -96,3 +110,23 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def warn_about_unsupported(self, values):
+        for value in values:
+            warn_once(
```

Comment:
You can hide it under the `filter_by` implementation -> Instead of calling `is_unsupported_float` directly you may wrap it with something like:
```python
def is_unsupported_float_with_warn(x):
    if is_unsupported_float(x):
        warn_once(...)
        return ....
    return ...
```

Reply:
Done!